scripts/crawl_expand.py

```diff
@@ -84,6 +83,7 @@ def makeNewStatusTable(config, updatedProviders, placeProviderIDs, newProviders)
 
 if __name__ == '__main__':
     TEST_CONFIG = { "yelp3": 3,
-                    "tripadvisor": 3 }
+                    "tripadvisor": 3,
+                    "google": 3 }
 
-    expandPlaces(TEST_CONFIG, GPS_LOCATIONS["CHICAGO_CENTER"], 3)
+    expandPlaces(TEST_CONFIG, GPS_LOCATIONS["CHICAGO_CENTER"], 30)
```

Comment:
Yep, I don't remember why I switched it to 3, bc the crawl script should by default crawl the 30 km circle of chicago - probably a mis-commit from before, so I'm undoing it now.